python/pfs/drp/stella/utils.py

```diff
@@ -188,3 +192,30 @@ def getPfsVersions(prefix="VERSION_"):
         importlib.import_module(module + ".version")
         versions[prefix + name] = sys.modules[module + ".version"].__version__
     return versions
+
+
+def headerToMetadata(header):
+    """Convert FITS header to LSST metadata
```

Comment:
Isn't this something we should move to `pfs_utils`?

Reply:
`pfs_utils` should know nothing about LSST.